### ISSUE TITLE
webview version of aria-flowto

### ISSUE THREAD

**Issue by annielovesjs** [microsoft/vscode#79059]
I am trying to control the behavior of the screen reader with the use of aria-flowto. It works on the browser but not on the webview on vscode. Is there an alternative to aria-flowto that could be used for webview?

**Comment by mjbvz** [comment]
Can you please some example extension code.

This is likely an upstream issue with electron/chromium, not with VS Code itself

**Comment by annielovesjs** [comment]
@mjbvz It works well with the browser (my project has a client version that we can serve on localhost for testing) but when I launched the extension on vscode and go through it with the screen reader, it seems aria-flowto does not work and the screen reader follows a different flow from the localhost version.

Example code of aria-flowto in my project (condensed for readability):
```
const Dropdown = ({
  id,
  ariaLabel,
  nextItem
}: IDropdownProps) => {
  return (
    <Select
      id={id}
      aria-label={ariaLabel}
      aria-flowto={nextItem}
    />
  );
};

<Dropdown  id={id} ariaLabel={ariaLabel} nextItem={nextItem}/>
```

**Comment by mjbvz** [comment]
This is not something VS Code can fix or really workaround. Please file an issue upstream [against electron](https://github.com/electron/electron). You can use [electron fiddle](https://github.com/electron/fiddle) for this. Make sure to test using the latest electron version too as it may already have been fixed.
